api/api.go

```diff
@@ -22,7 +22,7 @@ func (c *Client) req(method string, route string, data, resp interface{}) error
 		js, _ := json.Marshal(data)
 		body = bytes.NewReader(js)
 	}
-	req, err := http.NewRequest(method, fmt.Sprintf("%v%v", c.BaseURL, route), body)
+	req, err := http.NewRequest(method, fmt.Sprintf("%v/api%v", c.BaseURL, route), body)
```

Comment:
nah, not even here. Consider that nothing prevents someone from serving the API under a different prefix; `Client` would then be unable to access it. Also consider that, if you're making requests using `curl`, you'll manually specify the `/api` prefix; it would be confusing if that were not also the case for `Client`. It should be the caller's responsibility to pass the correct `BaseURL`.